xbmc/video/windows/GUIWindowVideoNav.h

```diff
@@ -49,7 +49,7 @@ class CGUIWindowVideoNav : public CGUIWindowVideoBase
   virtual bool OnAction(const CAction &action) override;
   virtual bool OnMessage(CGUIMessage& message) override;
 
-  virtual void OnItemInfo(const CFileItem& fileItem, ADDON::ScraperPtr &info) override;
+  virtual void OnItemInfo(CFileItem* fileItem, ADDON::ScraperPtr &info) override;
```

Comment:
FWIW, it was a pointer before https://github.com/xbmc/xbmc/commit/f9186f1b511450afae47ba5f5ccfb862ebc7d834

Maybe @Razzeee stumbled on a bug with the am commit?